cmd/repo.go

```diff
@@ -33,6 +35,22 @@ var addCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		spinner, _ := yacspin.New(util.SpinnerConf)
 
+		_, err := os.Stat(util.RepoPath)
+		if err != nil {
+			if os.IsNotExist(err) {
+				fmt.Printf("Repo path %s was not found: creating it\n ", util.RepoPath)
```

Comment:
I'm avoiding print calls because I'm going to add a logging package (that only shows output if you specify a `--verbose` flag

Reply:
What should I use instead?

Reply:
You can leave it out atm, I still need to decide what package I am going to use (or maybe I'm going to create something myself)